### DISCUSSION TITLE
Theme: Tokyo

### DISCUSSION THREAD

**Discussion by feamsr00** [JanDeDobbeleer/oh-my-posh#3401] [Troubleshoot]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### What happened?

A bug happened... Maybe

Is Tokyo supposed to have two root segments?

https://github.com/JanDeDobbeleer/oh-my-posh/blob/48797568869882ce33c622c4e1902b501d2a50c0/themes/tokyo.omp.json#L23-L30

https://github.com/JanDeDobbeleer/oh-my-posh/blob/48797568869882ce33c622c4e1902b501d2a50c0/themes/tokyo.omp.json#L69-L74

### Theme

tokyo.omp.json

### Which shell are you using?

powershell

### Log output

```Shell
N/A
```

**Comment by JanDeDobbeleer** [comment]
@feamsr00 don't know. That's a question for @daht-x, he can provide the answer. Definitely by design in this case so we can always tweak if needed but it's his theme.

**Reply by feamsr00** [reply]
@JanDeDobbeleer Small question. Why are there no author names on/with the themes? I could have made the issue on his repo or tagged him myself to be helpful.

**Reply by JanDeDobbeleer** [reply]
Depends on if they added this to the export. Wasn't required (nor isn't documented). All fine though, I don't mind doing that.
